modules/dag-shared/src/main/scala/org/tessellation/dag/package.scala

```diff
@@ -20,8 +20,8 @@ package object dag {
     classOf[DAGBlock] -> 603,
     DAGBlock.OrderingInstance.getClass -> 604,
     classOf[GlobalSnapshotInfo] -> 606,
-    classOf[DAGBlockAsActiveTip] -> 610,
-    DAGBlockAsActiveTip.OrderingInstance.getClass -> 611,
+    classOf[BlockAsActiveTip[_]] -> 610,
+//    BlockAsActiveTip.OrderingInstance.getClass -> 611,
```

Comment:
I am wondering if we can safely get rid of it. I was forced to implement `Ordering[BlockAsActiveTip]` as `def` so it works with any block but I am worried about backward compatibility 🤔

Reply:
@marcinwadon ^

Reply:
https://github.com/Constellation-Labs/tessellation/pull/546/files#diff-1d5cf500b56589e3429a0234cad7a312853d5e5972487c0eb1d783a682310b83

Reply:
Ok, I have confirmed that I need to keep it

Reply:
Yeah I think we can register both, e.i. ordering instance for `BlockAsActiveTip` parametrized by `DAGBlock` and we will register another ordering instance for `BlockAsActiveTip` parametrized by `CurrencyBlock`. Would that work?

Reply:
Ok I left both for now. We will see how it works when then Currency is implemented